<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>197</width>
         <height>448</height>
        </rect>
       </property>
       <attribute name="label">
        <string>图表设置</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>标题</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLineEdit" name="editTitle"/>
             </item>
             <item>
              <widget class="QPushButton" name="btnSetTitle">
               <property name="text">
                <string>设置</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="btnTitileFont">
             <property name="text">
              <string>设置标题字体</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>图例</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QRadioButton" name="radioButton">
             <property name="text">
              <string>上方</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QRadioButton" name="radioButton_2">
             <property name="text">
              <string>下方</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="radioButton_3">
             <property name="text">
              <string>左侧</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QRadioButton" name="radioButton_4">
             <property name="text">
              <string>右侧</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="radioChartVisible">
             <property name="text">
              <string>图例可见</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QRadioButton" name="radioBkVisible">
             <property name="text">
              <string>背景可见</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QPushButton" name="btnLegendFont">
             <property name="text">
              <string>字体</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="pushButton_5">
             <property name="text">
              <string>标签颜色</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>边距</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>左边距</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="spinMarginLeft"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>右边距</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spinMarginRight"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>上边距</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="spinMarginTop"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>下边距</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="spinMarginBottom"/>
           </item>
           <item row="4" column="0">
            <widget class="QPushButton" name="btnSetMargin">
             <property name="text">
              <string>设置边距</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>动画效果</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="1">
            <widget class="QComboBox" name="comboTheme"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>主题</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="comboAnimation"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>动画效果</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-42</y>
         <width>195</width>
         <height>354</height>
        </rect>
       </property>
       <attribute name="label">
        <string>曲线设置</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>选择操作序列</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QRadioButton" name="radioSeries0">
             <property name="text">
              <string>Sin曲线</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioSeries1">
             <property name="text">
              <string>Cos曲线</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>曲线外观</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0" colspan="2">
            <widget class="QPushButton" name="btnSeriesName">
             <property name="text">
              <string>序列名称</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="editSeriesName"/>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="chkSeriesVisible">
             <property name="text">
              <string>序列可见</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QCheckBox" name="chkPointVisible">
             <property name="text">
              <string>数据点可见</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QPushButton" name="btnSeriesColor">
             <property name="text">
              <string>曲线颜色</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="btnSeriesPen">
             <property name="text">
              <string>曲线Pen</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>透明度</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QSlider" name="sliderSeriesOpacity">
             <property name="maximum">
              <number>10</number>
             </property>
             <property name="pageStep">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_8">
          <property name="title">
           <string>数据点标签</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QCheckBox" name="chkPointLabelVisible">
             <property name="text">
              <string>数据点标签可见</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnDataLabelColor">
             <property name="text">
              <string>标签颜色</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnDataLabelFont">
             <property name="text">
              <string>标签字体</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_9">
          <property name="title">
           <string>标签格式</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QRadioButton" name="radioLabelFormy">
             <property name="text">
              <string>@yPoint</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioPLabelFormxy">
             <property name="text">
              <string>(@xPoint,@yPoint)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>-14</x>
         <y>-23</y>
         <width>194</width>
         <height>336</height>
        </rect>
       </property>
       <attribute name="label">
        <string>坐标轴设置</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGroupBox" name="groupBox_10">
          <property name="title">
           <string>设置坐标轴坐标</string>
          </property>
          <widget class="QRadioButton" name="radioX">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>73</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>x轴</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="radioY">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>30</y>
             <width>73</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>y轴</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_11">
          <property name="title">
           <string>坐标轴总体</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>最小值</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="spinAxisMin"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>最大值</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" rowspan="2">
            <widget class="QDoubleSpinBox" name="spinAxisMax"/>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkBox_4">
             <property name="text">
              <string>isVisible</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QPushButton" name="pushButton_11">
             <property name="text">
              <string>设置坐标范围</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>轴标题</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0" colspan="2">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>设置标题</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>设置字体</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="chkBoxAxisTitle">
             <property name="text">
              <string>isTitleVisible</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="editAxisTitle"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_12">
          <property name="title">
           <string>轴刻度标签</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="1">
            <widget class="QLineEdit" name="editAxisLabelFormat"/>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_12">
             <property name="text">
              <string>标签格式</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButton_14">
             <property name="text">
              <string>设置字体</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_13">
             <property name="text">
              <string>文字颜色</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="chkBoxLabelsVisble">
             <property name="text">
              <string>labelsVisible</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QChartView" name="chartView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
